### ISSUE TITLE
IDE storing the git repo somewhere else than the board

### ISSUE THREAD

**Issue by giuliomoro** [BelaPlatform/Bela#104]
This could be done but would require
1.`sshd` running on the host to allow inbound ssh connections
2. an ssh key pair that grants access to the the host to be stored on the board, which means that if the board is exposed on the internet and someone steals the key pair then the host is at potential risk

Leaving the security concerns to the user (whoever decides to start an sshd server on their machine better know what they are doing), and accounting at once for a git server running on the host machine or on a remote server (in case the user enabled internet sharing to the board), it would be useful to have a "git push" button.

Ultimately, it would be worth - instead - investigating the use of the github using the browser as a proxy (basically: do their servers support CORS? ).

**Comment by LBDonovan** [comment]
Yeah the security concerns outweigh any benefits to having a 'git push' on the board for me. I'm pretty sure on a mac at least you have to specifically allow ssh connections from Bela, and it's quite a pain to do.

The github stuff is probably worth looking into, but we would have to make sure people are aware of the potential privacy issues, in that they will be publishing their code publicly on github when doing this.
